d2l-button-icon.html

```diff
@@ -37,6 +37,10 @@
 			:host([h-align="text"]) button {
 				left: calc(((var(--d2l-button-icon-min-width) - 0.9rem) / 2) * -1);
 			}
+			:host-context([dir="rtl"])[h-align="text"] button {
+				left: 0;
+				right: calc(((var(--d2l-button-icon-min-width) - 0.9rem) / 2) * -1);
+			}
 			:host(:dir(rtl))[h-align="text"] button {
```

Comment:
`:host(:dir(rtl))[h-align="text"]` -> `:host([h-align="text"]:dir(rtl))`

Reply:
so this line is for P2/shady and it doesn't seem to work if I switch it to the option on the right.